packages/compartment-mapper/src/parse-cjs-shared-export-wrapper.js

```diff
@@ -176,10 +181,14 @@ export const wrap = ({
     if (exportsHaveBeenOverwritten) {
       moduleEnvironmentRecord.default = finalExports;
       for (const prop of keys(moduleEnvironmentRecord.default || {})) {
-        if (prop !== 'default')
+        if (prop !== 'default') {
           moduleEnvironmentRecord[prop] = moduleEnvironmentRecord.default[prop];
+        }
       }
     }
+    for (const prop of redefined) {
```

Comment:
there could be an `else` before this because redefined doesn't make a lot of sense when the default export was redefined. but it should also be an empty set in that case.